source/includes/read/distinct.cpp

```diff
@@ -0,0 +1,60 @@
+#include <iostream>
+
+#include <bsoncxx/builder/basic/document.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/client.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/uri.hpp>
+
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_document;
```

Comment:
```suggestion
using bsoncxx::builder::basic::make_document;
using bsoncxx::builder::basic::make_array;
```

To fix compile error:
```
distinct.cpp:54:61: error: use of undeclared identifier 'make_array'
                                                            make_array(make_document(kvp("borough", "Bronx")),
                                                            ^
```